tracing.go

```diff
@@ -252,10 +253,26 @@ func ExtractInboundSpan(ctx context.Context, call *InboundCall, headers map[stri
 
 func setPeerHostPort(span opentracing.Span, hostPort string) {
```

Comment:
Makes a lot of sense to avoid the parsing if we don't need it. If the `peer.hostPort` is not added, perhaps we should cache these values in the peer object so we don't need to reparse it constantly.